pkg/sql/catalog/systemschema/system.go

```diff
@@ -1756,6 +1764,49 @@ var (
 		FormatVersion:  descpb.InterleavedFormatVersion,
 		NextMutationID: 1,
 	})
+
+	// MigrationsTable is the descriptor for the migrations table. It stores facts
```

Comment:
PS I think this comment could do with fixing, it's referring to the table before.